Client/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift

```diff
@@ -0,0 +1,159 @@
+            profile: nil,
+            rewards: nil,
+            state: WelcomeViewCalloutState.defaultBrowserWarning(
+                title: Strings.Callout.defaultBrowserCalloutTitle,
+                details: Strings.Callout.defaultBrowserCalloutDescription,
+                primaryButtonTitle: Strings.Callout.defaultBrowserCalloutPrimaryButtonTitle,
+                secondaryButtonTitle: Strings.Callout.defaultBrowserCalloutSecondaryButtonTitle,
+                primaryAction: {
+                    guard let settingsUrl = URL(string: UIApplication.openSettingsURLString) else {
+                        return
+                    }
+                    
+                    Preferences.General.defaultBrowserCalloutDismissed.value = true
+                    UIApplication.shared.open(settingsUrl)
+                }, secondaryAction: { [weak self] in
+                    self?.dismiss(animated: false)
+                })
+        )
+
+        present(onboardingController, animated: true)
+        isfullScreenCalloutPresented = true
+    }
+    
+    func presentBraveRewardsScreenCallout() {
+        if Preferences.DebugFlag.skipNTPCallouts == true, isfullScreenCalloutPresented { return }
+
+        if !FullScreenCalloutManager.shouldShowDefaultBrowserCallout(calloutType: .rewards) {
+            return
+        }
+                
+        if BraveRewards.isAvailable, !Preferences.Rewards.rewardsToggledOnce.value {
+            let controller = OnboardingRewardsAgreementViewController(profile: profile, rewards: rewards)
+            controller.onOnboardingStateChanged = { [weak self] controller, state in
+                self?.dismissOnboarding(controller, state: state)
+            }
+            present(controller, animated: true)
+            isfullScreenCalloutPresented = true
+
+        }
+        return
+    }
+    
+    func presentSyncAlertCallout() {
+        if Preferences.DebugFlag.skipNTPCallouts == true, isfullScreenCalloutPresented { return }
+
+        if !FullScreenCalloutManager.shouldShowDefaultBrowserCallout(calloutType: .sync) {
+            return
+        }
+                
+        if !BraveSyncAPI.shared.isInSyncGroup {
+            var privacyEverywhereView = PrivacyEverywhereView()
+            let controller = PopupViewController(rootView: privacyEverywhereView)
+            
+            privacyEverywhereView.dismiss = { [unowned controller] in
+                controller.dismiss(animated: true)
+            }
+            
+            privacyEverywhereView.syncNow = { [unowned controller] in
+                controller.dismiss(animated: true) { [weak self] in
+                    guard let self = self else { return }
+                    self.openInsideSettingsNavigation(with: SyncWelcomeViewController())
+                }
+            }
+            
+            present(controller, animated: true, completion: nil)
+            isfullScreenCalloutPresented = true
+        }
+        
+        return
```

Comment:
this return can be removed

Reply:
Removed at https://github.com/brave/brave-ios/pull/4508/commits/429df1c4afce998ccc565b27120879a5afdfe87f